Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewController.swift

```diff
@@ -58,14 +58,6 @@ class WallpaperSelectorViewController: WallpaperBaseViewController, Themeable {
         return collectionView
     }()
 
-    private lazy var settingsButton: ResizableButton = .build { button in
-        button.titleLabel?.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .body,
-                                                                                size: 16)
-        button.titleLabel?.textAlignment = .center
-        button.setTitle(.Onboarding.WallpaperSelectorAction, for: .normal)
-        button.accessibilityIdentifier = AccessibilityIdentifiers.Onboarding.Wallpaper.settingsButton
```

Comment:
We should remove the AccessibilityIdentifier from `AccessibilityIdentifiers.swift` too.

Reply:
thanks for this too